spec/std/char_spec.cr

```diff
@@ -390,6 +390,15 @@ describe "Char" do
     '酒'.ascii?.should be_false
   end
 
+  it "#printable?" do
+    ' '.printable?.should be_true
+    'a'.printable?.should be_true
+    '酒'.printable?.should be_true
+    '\n'.printable?.should be_false
```

Comment:
I'd think that the newline character is included. It's certainly _visible_, and _printable_ - same as ASCII whitespace.

Reply:
I was thinking similar for tab.  I guess it depends what the purpose of this method is apart from what's needed in #11406